test/main.test.js

```diff
@@ -220,9 +233,9 @@ testMany('either', [
 
 testMany('coroutine', [
   () => {
-    const p = coroutine(function*() {
-      const firstPart = yield letters;
-      const secondPart = yield digits;
+    const p = coroutine(tokenize => {
```

Comment:
Same thing about calling the parameter yield.

Reply:
test was also updated